### PR TITLE
avm2: Add ContextMenu clipboard related properties

### PR DESCRIPTION
**Pull request by danielhjacobs**
Slightly progresses #10173.

Next error there is back to being TLF related.

### CODE REVIEW

**Review by Dinnerbone — APPROVED** [review]

File: core/src/avm2/globals/flash/ui/ContextMenu.as [hunk 1]
```diff
@@ -32,9 +35,25 @@ package flash.ui
...
         }
 
-        public static function get isSupported() : Boolean
-        {
+        public function get clipboardItems(): ContextMenuClipboardItems {
+            return this._clipboardItems;
+        }
+
+        public function set clipboardItems(value:ContextMenuClipboardItems):void {
+            this._clipboardItems = value;
```
Comment (Lord-McSweeney) [inline core/src/avm2/globals/flash/ui/ContextMenu.as:43]: This should have a stub warning?

File: core/src/avm2/globals/flash/ui/ContextMenu.as [hunk 2]
```diff
@@ -32,9 +35,25 @@ package flash.ui
...
+        }
+
+        public function get clipboardMenu():Boolean {
+            return _clipboardMenu;
+        }
+
+        public function set clipboardMenu(value:Boolean):void {
+            _clipboardMenu = value;
```
Comment (Lord-McSweeney) [inline core/src/avm2/globals/flash/ui/ContextMenu.as:51]: This should have a stub warning

File: core/src/avm2/globals/flash/ui/ContextMenu.as (hunk 1 above)
Comment (danielhjacobs) [reply core/src/avm2/globals/flash/ui/ContextMenu.as:43]: https://github.com/ruffle-rs/ruffle/pull/14038#discussion_r1395934331

Dinnerbone said that if things set what they should set, they shouldn't be stubs if if they do nothing. Difference in philosophy I guess. Probably a discussion for Discord.

Comment (Dinnerbone) [reply core/src/avm2/globals/flash/ui/ContextMenu.as:43]: Disagree, this code is doing what it's supposed to. The actual functionality is not here.

**Review by Lord-McSweeney — APPROVED** [review]

**Pull request merged**